templates/chiffee/purchase.html

```diff
@@ -0,0 +1,94 @@
+{% extends "chiffee/base.html" %}
+
+{% load chiffee_tags %}
+
+{% block body %}
+    <form action="{% url 'chiffee:make-purchases' %}" method="post">
+        {% csrf_token %}
+
+        <p>Wie viele willst du kaufen?</p>
+
+        {% for i in quantity_range %}
+            {% if i == quantity %}
```

Comment:
Can you put the if statement inside the button and just add the active with it?

Reply:
Done.